src/pages/SetPasswordPage.js

```diff
@@ -1,10 +1,8 @@
 import React, {Component} from 'react';
-import {
```

Comment:
Just a thought for a follow-up, we should add a custom lint rule to prevent usage of the `SafeAreaView` from `react-native`

Reply:
https://github.com/Expensify/eslint-config-expensify/pull/60